doc/help/man-ipptool.html

```diff
@@ -221,9 +221,9 @@ <h2 class="title"><a name="EXAMPLES">Examples</a></h2>
 </pre>
 <h2 class="title"><a name="SEE_ALSO">See Also</a></h2>
 <b>ipptoolfile</b>(5),
-IANA IPP Registry (<a href="http://www.iana.org/assignments/ipp\-registrations)">http://www.iana.org/assignments/ipp\-registrations)</a>,
+IANA IPP Registry (<a href="https:/www.iana.org/assignments/ipp-registrations">http://www.iana.org/assignments/ipp-registrations</a>),
```

Comment:
Missing a slash here...

Reply:
I'm sorry, but the changes you made in 33701600771 doesn't work in firefox - the closing brackets and the backslash in the links are the problem. If I remove them, the links are showed correctly.

The code currently tries to open these link:

```
https://www.iana.org/assignments/ipp\-registrations)
https://datatracker.ietf.org/doc/html/rfc8011)
```
The bracket shouldn't be there IMO.
If you wanted the brackets to be a part of the link label, it should be done this way (backslash in the text doesn't matter, but if we want to show the link which is under the text, then text should be the same):

```
<a href="https://www.iana.org/assignments/ipp-registrations)">(https://www.iana.org/assignments/ipp\-registrations)</a>
```

or if the only text inside the brackets should be a link:

```
(<a href="https://www.iana.org/assignments/ipp-registrations">https://www.iana.org/assignments/ipp\-registrations</a>)
```

Ad the backslash - I don't precisely recall the semantics of double quotes in html, but if it is the same as in bash, then the hyphen is taken as a literal, so it doesn't need to be escaped.

Reply:
OK, so the issue here is that mantohtml (the program that generates the HTML files from the man page sources) isn't handling the "\-" (literal "-" instead of endash) or recognizing the terminating parenthesis in the auto-detected links.